test/rotation.jl

```diff
@@ -0,0 +1,80 @@
+function rotation_testsuite(Backend)
+    device(x) = adapt(Backend(), x)
+    gradtest_fn = Backend == CPU ? gradtest : gputest
+    T = Float32
+    atol = T == Float32 ? 1e-3 : 1e-6
+    rtol = T == Float32 ? 1f-2 : 1f-6
```

Comment:
When I wrote the comment on relaxing the tolerances I hadn't seen that they were already high. So I suggest to restore it to 1f-3 or lower. Optimal would be to test where it fails and then set it a little higher than that.

Reply:
Seems like the tolerance works with 1f-3 now.